lntest/harness.go

```diff
@@ -671,7 +671,7 @@ func (n *NetworkHarness) EnsureConnected(ctx context.Context,
 // NOTE: This function may block for up to 15-seconds as it will not return
 // until the new connection is detected as being known to both nodes.
 func (n *NetworkHarness) ConnectNodes(ctx context.Context, t *testing.T,
-	a, b *HarnessNode) {
+	a, b *HarnessNode, perm bool) {
```

Comment:
nit: (no need to change) alternatively we could have `ConnectNodes(...)` and `ConnectNodesPerm(...)` both calling to `connectNodes(..., perm bool)`.